### DISCUSSION TITLE
[Feature]: Wakelock (keep screen on) in Backup screen / when performing "Start Backup"

### DISCUSSION THREAD

**Discussion by HcNguyen111** [immich-app/immich#1629] [Feature Request]
### Feature detail

Can a wakelock be added to the "Backup page", where the "Start Backup" button is to keep the phone screen on?

I had a look at [backup_controller_page.dart](https://github.com/immich-app/immich/blob/main/mobile/lib/modules/backup/views/backup_controller_page.dart) but am not sure how to add it.

It might be better to add a "Keep screen on" toggle/confirmation near "Start Backup" instead of forcing it for everyone all the time?

### Platform

Mobile App

**Comment by alextran1502** [comment]
Wake lock can be enabled from the device level in the screen lock setting.

**Comment by alextran1502** [comment]
Found a package can help with this 
https://pub.dev/packages/wakelock

**Comment by bo0tzz** [comment]
Wakelock is active on the backup page today (re-enabled via #22222 after being dropped during the drift migration). Closing.
